pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -178,6 +178,8 @@ def _pct(self, rate_by_ball):
         return '{0:.2f}'.format(rate_by_ball * 100)
 
     def _use_berry(self, berry_id, berry_count, encounter_id, catch_rate_by_ball, current_ball):
+        # Delay to simulate selecting berry
+        action_delay(self.config.action_wait_min, self.config.action_wait_max)
```

Comment:
I'd still like to humanize and add the delay for selecting a berry regardless of whether it has it's own request or not.

If it's considered unnecessary or unwanted I can remove or include an option for it?